td-shim-tools/src/enroller.rs

```diff
@@ -82,7 +83,10 @@ impl FirmwareRawFile {
         }
 
         // Update lengh field
```

Comment:
Not part of this PR, but a typo: "length".

Reply:
Fixed